code/src/.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
I am curious what the ```SKIP_PREFLIGHT_CHECK```  does? did you have incompatible versions of some of the dependencies? And if it was for unused packages maybe removing them is better ```npm uninstall name-of-package```